### PR TITLE
load device-migration-wizard JS in all places needed

### PR DESCRIPTION
**Pull request by escattone**
This PR does the following:
- a bit of renaming that I hope helps separate what the JS and SCSS files are for (the device migration wizard -- the HTML file is already named in this way [hook_device_migration_wizard.html](https://github.com/mozilla/kitsune/blob/switching-devices/kitsune/wiki/jinja2/wikiparser/hook_device_migration_wizard.html)) from where the JS bundle is needed (when displaying the content of the `switching-devices` KB article)
    - the `document.switching-devices.js` JS file → `device-migration-wizard.js`
    - the `document.switching-devices` JS bundle name → `switching-devices`
    - the `_switching-devices.scss` SCSS file → `_device-migration-wizard.scss`
- loads the `switching-devices` JS bundle in the additional places it's needed
- provides a function (which can also be used within Jinja templates) to check for a `switching-devices` document that accounts for translations (so this PR replaces my PR mozilla/kitsune#5460)

### CODE REVIEW

File: kitsune/wiki/templatetags/jinja_helpers.py [hunk 1]
```diff
@@ -16,3 +17,10 @@ def diff_table(content_from, content_to):
 @library.global_function
 def generate_video(v):
     return Markup(parser.generate_video(v))
+
+
+@library.global_function
+def is_switching_devices_document(doc):
+    return (
+        doc.parent.slug if doc.parent else doc.slug
```
Comment (akatsoulas) [inline kitsune/wiki/templatetags/jinja_helpers.py:25]: shall we make this a property on the model?

**Review by akatsoulas — APPROVED** [review]
r+[wc]?

**Review by mikeconley — APPROVED** [review]
This works for me! Thanks!

File: kitsune/wiki/templatetags/jinja_helpers.py (hunk 1 above)
Comment (escattone) [reply kitsune/wiki/templatetags/jinja_helpers.py:25]: Yes, that's better, thanks @akatsoulas.

**Pull request merged**